composer/trainer/_patch_pytorch.py

```diff
@@ -1101,3 +1106,52 @@ def unshard_with_sync(self):
             raise RuntimeError('CUDA out of memory encountered on a different rank')
         padded_unsharded_flat_param = self._all_gather_flat_param(unsharded_flat_param)
         self._use_unsharded_flat_param(padded_unsharded_flat_param)
+
+if version.parse(torch.__version__) >= version.parse('2.6.0') and version.parse(
```

Comment:
is this an exact copy of the patch above? should the range of that patch just be expanded to include 2.6?

Reply:
Oh yeah that's right, I just misread the code from earlier.